base/operators.jl

```diff
@@ -940,9 +940,9 @@ function ∘ end
 
 Represents the composition of two callable objects `outer::Outer` and `inner::Inner`. That is
 ```julia
-ComposedFunction(outer, inner)(args...; kw...) === outer(inner(args...; kw...))
+ComposedFunction(outer, inner)(args...; kw...) === outer(inner(args...; kw...); kw...)
```

Comment:
I don't think this is the right definition. Arguments to the composed function should only be passed to `inner`. If it were possible for `inner` to return something that caused keyword arguments to be passed to `outer`, then that would be the right thing to do. Since that isn't possible, the outer function should never be passed keyword arguments.

The situation is similar to that with varargs: the inner function can take any number of arguments, but the outer function can only accept the inner function's returned value.